src/Modules/Settings.php

```diff
@@ -244,6 +244,9 @@ public function one_tap_login(): void {
 					value='1'>
 			<?php esc_html_e( 'One Tap Login', 'login-with-google' ); ?>
 		</label>
+		<p>
+			<?php esc_html_e( 'Warning: One Tap login is more convenient, but it bypasses two-factor authentication (2FA).', 'login-with-google' ); ?>
```

Comment:
Since, it's a warning we may want to change the color of this message probably to something red or yellow.

Reply:
When I added `error-message` class to the `<p>`, it looks similar to this 

<img width="1239" height="654" alt="2025-11-18_14-22" src="https://github.com/user-attachments/assets/a40bb1f2-c52d-4308-979f-7d0c3fbd09df" />